server/monitor/src/main/java/org/apache/accumulo/monitor/rest/trace/TracesResource.java

```diff
@@ -77,12 +81,12 @@
    * Generates a trace summary
    *
    * @param minutes
-   *          Range of minutes to filter traces
+   *          Range of minutes to filter traces Min of 0 minutes, Max of 30 days
    * @return Trace summary in specified range
    */
   @Path("summary/{minutes}")
   @GET
-  public RecentTracesList getTraces(@DefaultValue("10") @PathParam("minutes") int minutes) throws Exception {
+  public RecentTracesList getTraces(@DefaultValue("10") @PathParam("minutes") @Min(0) @Max(2592000) int minutes) throws Exception {
```

Comment:
Presumably, we'd configure the monitor's servlet properly so that Jersey enforces them. This PR does not yet have those bits.